docs/explanation/launchpad.rst

```diff
@@ -39,12 +39,12 @@ or similar platforms.
 Personal Package Archive (PPA)
 ------------------------------
 
-Launchpad PPAs allow you to build installable Ubuntu packages for multiple
+Launchpad PPA repositories allow you to build installable Ubuntu packages for multiple
 :term:`architectures <Architecture>` and to host them in your own software
 :term:`repository <Repository>`. 
```

Comment:
note: A bit redundant in the context of the remaining sentence.

Reply:
The main motivation is to avoid the use of plural acronyms.